conf/messages.cy

```diff
@@ -146,6 +146,8 @@ paymentsHistory.previousPayments.youCan = Gallwch
 paymentsHistory.previousPayments.viewPreviousPayments = fwrw golwg dros eich taliadau blaenorol (yn agor tab newydd)
 paymentsHistory.previousPayments.beforeMtd = os gwnaethoch daliadau cyn ymuno â’r cynllun Troi Treth yn Ddigidol.
 
+vatCertificate.title = Your VAT Certificate
```

Comment:
Eich tystysgrif TAW

I know it's not the point lol but just in case it gets missed.